Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -65,9 +64,9 @@ public func assertSnapshots<Value, Format>(
   line: UInt = #line
   ) {
 
-  strategies.forEach { name, strategy in
+  try? strategies.forEach { name, strategy in
```

Comment:
should we make `assertSnapshots` throw? on this cause any `value: () -> Value` throws to be swallowed?